src/components/__tests__/Checkbox/CheckboxItem.test.tsx

```diff
@@ -94,3 +94,13 @@ it('disables the row when the prop disabled is true', () => {
     accessibilityState: { disabled: true },
   });
 });
+
+it('should have maxFontSizeMultiplier set to 1.5 by default', () => {
+  const { getByTestId } = render(
+    <Checkbox.Item label="" testID="checkbox-item" status="unchecked" />
+  );
+  const checkboxItem = getByTestId('checkbox-item');
+  expect(
+    checkboxItem.props.children[0].props.children[1].props.maxFontSizeMultiplier
```

Comment:
Add a `testID` to the `Text` in the checkbox.
```diff
         <Text
           variant={labelVariant}
           maxFontSizeMultiplier={maxFontSizeMultiplier}
+          testID={`${testID}-text`}
```

And then update the test

```suggestion
  const checkboxItemText = getByTestId('checkbox-item-text');
  expect(
    checkboxItemText.props.maxFontSizeMultiplier
```